src/GraphQL/Conversion/DateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,81 @@
+// <copyright file="DateTimeOffsetConverter.cs" company="wut">
+//   Copyright © wut
+// </copyright>
```

Comment:
Need to remove this copyright.

Reply:
Sure, accidentally copy-pasted it from main solution.